client/hosting/overview/components/plan-bandwidth.tsx

```diff
@@ -22,9 +36,22 @@ const getCurrentMonthRangeTimestamps = () => {
 };
 
 export function PlanBandwidth( { siteId }: PlanBandwidthProps ) {
+	const [ bandwidthData, setBandwidthData ] = useState< SiteMetricsAPIResponse >();
 	const selectedSiteData = useSelector( getSelectedSite );
+	const siteSlug = useSelector( ( state ) => getSiteSlug( state, siteId ) );
+	const isAtomic = useSelector( ( state ) => isAtomicSite( state, siteId ) );
+	const canViewStat = useSelector( ( state ) => canCurrentUser( state, siteId, 'publish_posts' ) );
+	const { isSiteAtomic, hasSftpFeature } = useSelector( ( state ) => ( {
+		isSiteAtomic: isSiteWpcomAtomic( state, siteId as number ),
+		hasSftpFeature: siteHasFeature( state, siteId, FEATURE_SFTP ),
+	} ) );
```

Comment:
```suggestion
	const isSiteAtomic = useSelector( ( state ) => isSiteWpcomAtomic( state, siteId as number ) );
	const hasSftpFeature = useSelector( ( state ) => siteHasFeature( state, siteId, FEATURE_SFTP ) );
```
Destructuring seems redundant here and it's unclear why we treat these variables differently from the ones above. I would just use two separate `useSelector` calls.

Reply:
👍 Thanks! Looking back, these wound up being treated separately because I added this logic after the initial variables, and was emulating [the condition used to determine whether or not to display an activation button for hosting features](https://github.com/Automattic/wp-calypso/blob/4d73c032eb95cbd50ebae7a223a74e4b895560a5/client/hosting/hosting-features/components/hosting-features.tsx#L47-L48). Since we're matching that state (ready to go atomic) I wanted to use consistent logic, and I wound up carrying the destructuring over from there. You're right though, it's more consistent and makes more sense with your suggestion.